src/components/games/rps.ts

```diff
@@ -323,11 +324,7 @@ export type RpsGameState = {
   player2Sign: RpsGameSign;
 };
 
-function getRandomInt(max: number) {
-  return Math.floor(Math.random() * max);
-}
-
 // Algorithm to get RPS game sign for Codey
 export const getCodeyRpsSign = (): RpsGameSign => {
-  return 1 + getRandomInt(3);
+  return getRandomInt(3);
 };
```

Comment:
I think we should keep this to be `1 + getRandomInt(3)`, if we follow my comment in `utils/num.ts`.